TestScene/TestScene.cs

```diff
@@ -3,15 +3,15 @@
 
 public class TestScene : Node2D
 {
-    // Declare member variables here. Examples:
-    // private int a = 2;
-    // private string b = "text";
```

Comment:
I noticed that spaces are being changed to tabs. We should probably try to stay consistent with that. Are you opposed to spaces to stick with what my code has used so far?

@alecMcgrail your thoughts welcome here too ^